### PR TITLE
Feature/youtube social link

### PR DESCRIPTION
**Pull request by pixel-glyph**
YouTube social icon added in footer of page. 
**NOTE** - Remove style.css.map from master before merging.

screen shot 2016-06-24 at 12 25 53 pm

### CODE REVIEW

File: easydita-knowledge-portal/functions.php [hunk 1]
```diff
@@ -128,7 +128,7 @@ function easydita_knowledge_portal_scripts() {
 function easydita_knowledge_portal_change_style_version_num($styles) {
 	$styles -> default_version = "123";
 }
-//add_action("wp_default_styles", "easydita_knowledge_portal_change_style_version_num");
+add_action("wp_default_styles", "easydita_knowledge_portal_change_style_version_num");
```
Comment (slucchini) [inline easydita-knowledge-portal/functions.php]: This shouldn't get committed, by default we want the basic caching that WP uses for the style.css file.

File: easydita-knowledge-portal/footer.php [hunk 2]
```diff
@@ -33,6 +33,9 @@
 	  			<?php if (get_theme_mod('linkedin_enabled',1) == 1) : ?>
 	  				<a href="<?php echo get_theme_mod('linkedin_link','#') ?>" target="_blank"><i class="linkedin-icon"></i></a>
 	  			<?php endif ?>
+					<?php if (get_theme_mod('youtube_enabled',1) == 1) : ?>
```
Comment (slucchini) [inline easydita-knowledge-portal/footer.php:36]: Too many tabs?

File: easydita-knowledge-portal/functions.php [hunk 3]
```diff
@@ -148,9 +148,9 @@ function easydita_knowledge_portal_get_hierarchy() {
 		$hierarchy = array();
 		if($post->ID) {
 			$hierarchy[] = $post->ID;
-
```
Comment (slucchini) [inline easydita-knowledge-portal/functions.php:151]: Again, be careful about what you stage and commit. You should use `git diff --staged` before committing each time to make sure you don't have any unnecessary changes (like these whitespace issues).

Comment (pixel-glyph) [inline easydita-knowledge-portal/functions.php:151]: Hmm, this didn't show up in my local diff.

**Comment by direkitteh** [comment]
lgtm

File: easydita-knowledge-portal/functions.php (hunk 3 above)
Comment (slucchini) [inline easydita-knowledge-portal/functions.php:151]: Huh, interesting. Ok, well maybe we can figure out how to avoid this in the future, but no worries.

File: easydita-knowledge-portal/sass/ezd/footer.scss [hunk 4]
```diff
@@ -58,17 +58,20 @@
 	i.linkedin-icon:after {
 		content: "\f08c"
 	}
+  i.youtube-icon:after {
+		content: "\f167"
```
Comment (slucchini) [inline easydita-knowledge-portal/sass/ezd/footer.scss]: Let's use \f166 as the default here (with the background: http://fontawesome.io/icon/youtube-square/) so that it matches the styling of the other icons in the default skin. Then, you can use \f167 in the skin1/footer.css file to overwrite the icon for the new skin.

File: easydita-knowledge-portal/functions.php (hunk 1 above)
Comment (pixel-glyph) [inline easydita-knowledge-portal/functions.php]: Huh, don't recall doing that.

File: easydita-knowledge-portal/footer.php (hunk 2 above)
Comment (pixel-glyph) [inline easydita-knowledge-portal/footer.php:36]: This also isn't showing locally. Something strange is afoot!

File: easydita-knowledge-portal/functions.php (hunk 3 above)
Comment (pixel-glyph) [inline easydita-knowledge-portal/functions.php:151]: Discovered what was causing the issue: my text editor removes trailing whitespace by default.

**Comment by pixel-glyph** [comment]
Changes have been made.

File: easydita-knowledge-portal/footer.php (hunk 2 above)
Comment (direkitteh) [inline easydita-knowledge-portal/footer.php:36]: I bet they're tabs instead of spaces ;)
